src/content/docs/en/pages/guides/marketplace/templates/stencil-boilerplate.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: How to deploy edge applications with the Stencil Boilerplate
+description: >-
+  The Stencil Boilerplate provides an automation solution for deploying a Stencil
+  application on Azion.
+meta_tags: Stencil, templates, guides, Azion Marketplace
+namespace: docs_guides_stencil_boilerplate
+permalink: /documentation/products/guides/stencil-boilerplate/
+---
+
+import Tag from 'primevue/tag';
+
+<Tag severity="info" client:only="vue">
+Preview
+</Tag>
+
+With the **Stencil Boilerplate**, you can start a project using [Stencil](https://stenciljs.com/docs/introduction) and deploy it at the edge. **Stencil** is a compiler that generates Web Components and combines the best concepts of the most popular frameworks into a simple build-time tool.
+
+The deployment of this template creates a GitHub repository containing your project, as well as an edge application and a domain to facilitate your access and management through Azion Edge Platform.
+
+This template uses Stencil version `4.22.2`.
+
+---
+
+## Requirements
+
+- A [GitHub account](https://github.com/signup) to connect with Azion and create your new repository.
+    - Every push will be deployed automatically to this repository to keep your project updated.
+- This template uses [Application Accelerator](/en/documentation/products/build/edge-application/application-accelerator/), [Edge Functions](/en/documentation/products/build/edge-application/edge-functions/), and [Edge Cache](https://www.azion.com/en/documentation/products/build/edge-application/edge-cache/), and it could generate usage-related costs. Check the [pricing page](https://www.azion.com/en/pricing/) for more information.
```

Comment:
```suggestion
- This template uses [Application Accelerator](/en/documentation/products/build/edge-application/application-accelerator/), [Edge Functions](/en/documentation/products/build/edge-application/edge-functions/), and [Edge Cache](/en/documentation/products/build/edge-application/edge-cache/), and it could generate usage-related costs. Check the [pricing page](https://www.azion.com/en/pricing/) for more information.
```